src/main/java/org/matsim/run/rebalancing/WaitingPointBasedRebalancingStrategyParams.java

```diff
@@ -0,0 +1,42 @@
+/* *********************************************************************** *
+ * project: org.matsim.*
+ * *********************************************************************** *
+ *                                                                         *
+ * copyright       : (C) 2018 by the members listed in the COPYING,        *
+ *                   LICENSE and WARRANTY file.                            *
+ * email           : info at matsim dot org                                *
+ *                                                                         *
+ * *********************************************************************** *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *   See also COPYING, LICENSE and WARRANTY file                           *
+ *                                                                         *
+ * *********************************************************************** */
+
+package org.matsim.run.rebalancing;
+
+import jakarta.validation.constraints.NotNull;
+import org.matsim.contrib.drt.optimizer.rebalancing.RebalancingParams;
+import org.matsim.core.config.ReflectiveConfigGroup;
+
+/**
+ * @author Chengqi Lu
+ * // TODO this is not needed for Kelheim scenario. I will move it to the matsim-lib after eeverything works well here.
+ */
+public final class WaitingPointBasedRebalancingStrategyParams extends ReflectiveConfigGroup
+		implements RebalancingParams.RebalancingStrategyParams {
+	public static final String SET_NAME = "waitingPointBasedRebalancingStrategy";
+
+	@Parameter("waiting points of the vehicle")
+	@Comment("The path to the waiting point file (csv/tsv) can be specified here. title row of the file: link_id	capacity" +
+		"If unspecified (i.e., empty string by default), starting points of the fleet will be used as the waiting points")
```

Comment:
how does this work. The code in `RunKelheimScenario` reads like this rebalancing strategy would not be installed, if no waitin points file path wass provided. `WaitingPointsBasedRebalancingStrategy.initialize()` also seems to assume to always read in a `.tsv` file. 
Am i missing something?
Actually, you are right: For Kelheim, it would be convenient to use the start locations, as we have spreaded them throughout the corresponding service areas with some thoughts (they represent the waiting points for the realistic fleet sizes). Also, we would have to come up with one file per fleet size, because the current version throws an error if the total capacity is insufficent. That is not a major problem, of course.